<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigSystem</class>
 <widget class="QWidget" name="frmConfigSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetLeft" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gboxBase">
        <property name="title">
         <string>常规设置 - 立即应用</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labAutoRun">
           <property name="text">
            <string>开机运行</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="SwitchButton" name="btnAutoRun" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labTitleC">
           <property name="text">
            <string>中文标题</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" colspan="3">
          <widget class="QLineEdit" name="txtTitleCn"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labAutoLogin">
           <property name="text">
            <string>自动登录</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="SwitchButton" name="btnAutoLogin" native="true"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labTitleE">
           <property name="text">
            <string>英文标题</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="3">
          <widget class="QLineEdit" name="txtTitleEn"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labAutoPwd">
           <property name="text">
            <string>记住密码</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="SwitchButton" name="btnAutoPwd" native="true"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labCopyright">
           <property name="text">
            <string>版权所有</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="3">
          <widget class="QLineEdit" name="txtCopyright">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labSaveLog">
           <property name="text">
            <string>调试日志</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="SwitchButton" name="btnSaveLog" native="true"/>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="labWorkMode">
           <property name="text">
            <string>工作模式</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QComboBox" name="cboxWorkMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="labNavStyle">
           <property name="text">
            <string>导航样式</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QComboBox" name="cboxNavStyle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labSaveRunTime">
           <property name="text">
            <string>运行时间</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="SwitchButton" name="btnSaveRunTime" native="true"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="labStyleName">
           <property name="text">
            <string>界面样式</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QComboBox" name="cboxStyleName"/>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="labLogoImage">
           <property name="text">
            <string>软件图标</string>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QComboBox" name="cboxLogoImage"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labFullScreen">
           <property name="text">
            <string>全屏模式</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="SwitchButton" name="btnFullScreen" native="true"/>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="labTipInterval">
           <property name="text">
            <string>弹框时间</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QComboBox" name="cboxTipInterval"/>
         </item>
         <item row="5" column="4">
          <widget class="QLabel" name="labAlarmSoundCount">
           <property name="text">
            <string>报警声音</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QComboBox" name="cboxAlarmSoundCount"/>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="labTimeHideCursor">
           <property name="text">
            <string>鼠标隐藏</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QComboBox" name="cboxTimeHideCursor"/>
         </item>
         <item row="6" column="4">
          <widget class="QLabel" name="labTimeAutoFull">
           <property name="text">
            <string>自动全屏</string>
           </property>
          </widget>
         </item>
         <item row="6" column="5">
          <widget class="QComboBox" name="cboxTimeAutoFull"/>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="labMsgListCount">
           <property name="text">
            <string>图文警情</string>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QComboBox" name="cboxMsgListCount"/>
         </item>
         <item row="7" column="4">
          <widget class="QLabel" name="labMsgTableCount">
           <property name="text">
            <string>窗口信息</string>
           </property>
          </widget>
         </item>
         <item row="7" column="5">
          <widget class="QComboBox" name="cboxMsgTableCount"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labRightInfo">
           <property name="text">
            <string>右上信息</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="SwitchButton" name="btnRightInfo" native="true"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labSaveUrl">
           <property name="text">
            <string>播放记录</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="SwitchButton" name="btnSaveUrl" native="true"/>
         </item>
         <item row="8" column="3" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="XSlider" name="sliderWindowOpacity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labWindowOpacityValue">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="labWindowOpacity">
           <property name="text">
            <string>透明度值</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="labUse96Dpi">
           <property name="text">
            <string>屏幕缩放</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="SwitchButton" name="btnUse96Dpi" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxColor">
        <property name="title">
         <string>颜色配置 - 立即应用</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="2">
          <widget class="QLabel" name="labColorTextWarn">
           <property name="text">
            <string>警告文字颜色</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labColorIconAlarm">
           <property name="text">
            <string>报警图标颜色</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="ColorComboBox" name="cboxColorIconAlarm"/>
         </item>
         <item row="1" column="1">
          <widget class="ColorComboBox" name="cboxColorTextAlarm"/>
         </item>
         <item row="0" column="3">
          <widget class="ColorComboBox" name="cboxColorTextWarn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ColorComboBox" name="cboxColorTextNormal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labColorTextNormal">
           <property name="text">
            <string>正常文字颜色</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labColorTextAlarm">
           <property name="text">
            <string>报警文字颜色</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labColorMsgWait">
           <property name="text">
            <string>等待处理颜色</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ColorComboBox" name="cboxColorMsgWait"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labColorMsgSolved">
           <property name="text">
            <string>已经处理颜色</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="ColorComboBox" name="cboxColorMsgSolved"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxDb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>数据库设置 - 重启生效</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="frmConfigDb" name="widgetConfigDb" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widgetRight" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>视频参数 - 立即应用</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="frmConfigVideo" name="widgetConfigVideo" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxEnable">
        <property name="title">
         <string>功能激活 - 立即应用或重启生效</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="frmConfigEnable" name="widgetConfigEnable" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>718</width>
       <height>230</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XSlider</class>
   <extends>QSlider</extends>
   <header>xslider.h</header>
  </customwidget>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">colorcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>switchbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>frmConfigDb</class>
   <extends>QWidget</extends>
   <header>frmconfigdb.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>frmConfigEnable</class>
   <extends>QWidget</extends>
   <header>frmconfigenable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>frmConfigVideo</class>
   <extends>QWidget</extends>
   <header>frmconfigvideo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtTitleCn</tabstop>
  <tabstop>txtTitleEn</tabstop>
  <tabstop>txtCopyright</tabstop>
  <tabstop>cboxWorkMode</tabstop>
  <tabstop>cboxNavStyle</tabstop>
  <tabstop>cboxStyleName</tabstop>
  <tabstop>cboxLogoImage</tabstop>
  <tabstop>cboxTipInterval</tabstop>
  <tabstop>cboxAlarmSoundCount</tabstop>
  <tabstop>cboxTimeHideCursor</tabstop>
  <tabstop>cboxTimeAutoFull</tabstop>
  <tabstop>cboxMsgListCount</tabstop>
  <tabstop>cboxMsgTableCount</tabstop>
  <tabstop>sliderWindowOpacity</tabstop>
  <tabstop>cboxColorTextNormal</tabstop>
  <tabstop>cboxColorTextWarn</tabstop>
  <tabstop>cboxColorTextAlarm</tabstop>
  <tabstop>cboxColorIconAlarm</tabstop>
  <tabstop>cboxColorMsgWait</tabstop>
  <tabstop>cboxColorMsgSolved</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
